plugins/TwoFactorAuth/lang/en.json

```diff
@@ -2,50 +2,50 @@
+        "TwoFactorAuthenticationRequired": "Two-factor authentication is required for everyone, you cannot turn it off.",
+        "ConfigureDifferentDevice": "Set up a new device",
         "SetUpTwoFactorAuthentication": "Set up two-factor authentication (2FA)",
-        "RequiredToSetUpTwoFactorAuthentication": "You are required to set up two-factor authentication before you can log in",
+        "RequiredToSetUpTwoFactorAuthentication": "You must set up two-factor authentication before logging in.",
         "AuthenticationCode": "Authentication code",
         "Verify": "Verify",
         "StepX": "Step %s",
-        "MissingAuthCodeAPI": "Please specify two-factor authentication code.",
-        "InvalidAuthCode": "The two-factor authentication code is not correct.",
-        "RequiredAuthCodeNotConfiguredAPI": "You are required to set up two-factor authentication. Please log in to your account.",
-        "VerifyIdentifyExplanation": "Open the two-factor authentication app on your device to view your authentication code and verify your identity.",
-        "DontHaveYourMobileDevice": "Don’t have your mobile device?",
+        "MissingAuthCodeAPI": "Please specify a two-factor authentication code.",
+        "InvalidAuthCode": "Incorrect two-factor authentication code.",
+        "RequiredAuthCodeNotConfiguredAPI": "You must set up two-factor authentication. Please log in to do so.",
+        "VerifyIdentifyExplanation": "Verify yourself by opening the two-factor authentication app on your device to get your authentication code.",
+        "DontHaveYourMobileDevice": "Don't have your mobile device?",
         "EnterRecoveryCodeInstead": "Enter one of your recovery codes",
-        "AskSuperUserResetAuthenticationCode": "Ask super user to reset your authentication code",
+        "AskSuperUserResetAuthenticationCode": "Ask a superuser to reset your authentication code",
         "SetupIntroFollowSteps": "Please follow these steps to set up two-factor authentication:",
-        "SetupFinishedTitle": "Congratulations! Your account is now more secure.",
-        "SetupFinishedSubtitle": "You have successfully set up two-factor authentication. Next time you log in, you will need to also enter the authentication code. Make sure you have your mobile device or your backup codes with you.",
-        "WarningChangingConfiguredDevice": "You are about to change the configured two-factor authentication device. This will invalidate any previously configured device.",
+        "SetupFinishedTitle": "Your account is now more secure.",
+        "SetupFinishedSubtitle": "You have now set up two-factor authentication. From now on you nee to enter an authentication code to log in. Make sure you have your mobile device or your backup codes with you.",
+        "WarningChangingConfiguredDevice": "You are about to change the configured two-factor authentication device. This will invalidate any previously configured devices.",
```

Comment:
It's only possible to configure one device at a time, so `device` would be correct here. But if you think `devices` would be a better English that should be fine as well.